.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Why the changes were made
+
+<!-- Explain why this PR is important, what problems it fixes, link related issues -->
+
+## How the changes were made
```

Comment:
Personally I don't think this is a clear question..

The upstream as the second question has: "Does your change fix a particular issue?" which is NOT required if the PR relates to a jira issue as it often does.   Perhaps we can repurpose to say...
"If there is no Jira, what issue is being fixed with the PR" or something like that.

Reply:
"Link to origin of request", slack, jira, github, gitlab, reddit, google meet, google doc.

Reply:
Here I do not want to know about issues (WHY), added issue part to the first section (we can add more description to it)

Here I want to know HOW, what was done to achieve the goal, which commands were run, what files were modified, what possibilities were considered, etc

Reply:
I think this feedback would be for the why section :)

Reply:
removed